tests/test-nullspace.C

```diff
@@ -157,24 +192,21 @@ int main(int argc, char** argv)
     }
 
     srand(seed);
+    std::cout << "With seed: " << seed << std::endl;
```

Comment:
Not sure I like to display the seed here only: when running in loop mode, it is relevant to display the seed generating the last iteration that caused the failure. Maybe, print at the endd `seed-1` ?